### ISSUE TITLE
Add GraphQL Subscription Example

### ISSUE THREAD

**Issue by chuck-alt-delete** [MaterializeInc/connection-examples#25]
Related to #10 . It seems like [GraphQL subscriptions and Materialize fit like hand in glove](https://www.apollographql.com/docs/react/data/subscriptions/#when-to-use-subscriptions). We should add an example GraphQL API that allows end users to subscribe to views in Materialize.

**Comment by benesch** [comment]
Indeed, although the tricky part is writing the backend code that converts an arbitrary GraphQL subscription to the appropriate Materialize `SUBSCRIBE` command while handling auth, etc.

**Comment by benesch** [comment]
One thing to note: GraphQL subscriptions aren't a standardized thing. In particular they don't offer any help with update framing—no specification of e.g. "this is how you represent a delete vs an insert vs an update." Every client and server has a different take on how to communicate updates. So we'd have to invent or repurpose one of the existing protocols for this.

**Comment by chuck-alt-delete** [comment]
Turns out this was exactly what I was looking for:
- https://github.com/MaterializeInc/demos/tree/main/antennas-postgres/backend

this opens up a new question for me about how to better surface both connection-examples and demos repos in the docs to to make it much easier for someone to search something like “materialize graphql” in the docs and get to that code.
